.github/workflows/python.yaml

```diff
@@ -22,7 +29,29 @@ jobs:
         pip install flake8 pytest
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
```

Comment:
I knew I was missing something!  :D

Since the requirements.txt is just for testing requirements, I think it makes sense to remove it outright and when we need actual runtime dependencies later, re-add and always install from it later.